.github/workflows/hawkscan-integration.yml

```diff
@@ -0,0 +1,31 @@
+name: HawkScan Integration Tests
+on:
+  repository_dispatch:
+    types: [integration-test]
+jobs:
+  hawkscan:
+    name: HawkScan Action Integration Test
+    runs-on: ubuntu-latest
```

Comment:
Consider using ubuntu-20.04 to avoid problems in the future when ubuntu-latest points to a new Ubuntu release.